packages/token-lists/src/tokens/pancakeswap-top-100.json

```diff
@@ -0,0 +1,814 @@
+{
+  "name": "PancakeSwap Top 100",
+  "timestamp": "2021-03-17T09:56:23Z",
+  "version": {
+    "major": 2,
+    "minor": 10,
+    "patch": 0
+  },
+  "tags": {},
+  "logoURI": "https://exchange.pancakeswap.finance/images/pancakeswap.png",
+  "keywords": ["pancake", "top 100"],
+  "tokens": [
```

Comment:
This file should just contains tokens, i.e. stuff from `[` to `]` here, nothing else